src/main/java/com/hyperwallet/clientsdk/model/HyperwalletBusinessStakeholder.java

```diff
@@ -640,4 +642,26 @@ public HyperwalletBusinessStakeholder clearCountry() {
         country = null;
         return this;
     }
+
+    public List<HyperwalletLink> getLinks() {
+        return links;
+    }
+
+    public void setLinks(List<HyperwalletLink> links) {
+        addField("links", links);
+        this.links = links;
+    }
+
+    public HyperwalletBusinessStakeholder links(List<HyperwalletLink> links) {
```

Comment:
I see that this method is a composition of setLinks method and a return statement. I notice this way of writing everywhere. See if you can call setLinks instead of lines 656 and 657. This is a suggestion.